src/components/modules/blockEvents.ts

```diff
@@ -252,14 +252,9 @@ export default class BlockEvents extends Module {
       return;
     }
 
-    /**
-     * The Toolbox will be opened with immediate focus on the Search input,
-     * and '/' will be added in the search input by default — we need to prevent it and add '/' manually
-     */
-    event.preventDefault();
-    this.Editor.Caret.insertContentAtCaretPosition('/');
-
-    this.activateToolbox();
+    setTimeout(() => {
+      this.activateToolbox();
+    }, 0);
```

Comment:
1) this change looks unclear for me. Previous solutuion looks more explicit. Why it is not working?

2) We're trying to get rid of `setTimeout` in a code since it leads of problems of caret behavior which are hard to maintain.

Reply:
The reason the toolbox cannot be invoked with the Chinese input method / is that the Chinese / corresponds to `event.key = Process` instead of `/`, so `event.code === 'Slash'` is used to determine if / is pressed. Using `setTimeout` to invoke the toolbox is mainly to asynchronously invoke the toolbox because the original code's `event.preventDefault()` cannot prevent the / input in Chinese mode, causing / to be automatically entered in the `SearchInput`. I don't have any ideas for better asynchronous calls yet.

Reply:
> the original code's `event.preventDefault()` cannot prevent the / input in Chinese mode, causing / to be automatically entered in the `SearchInput`. 

So how it prevented now?